src/background.js

```diff
@@ -133,7 +133,7 @@ const messageOps = {
     delete message.tabId;
     toSave[idForItem(message)] = message;
     return browser.tabs.query({}).then(tabs => {
-      if (tabs.length <= 1) {
+      if (tabs.length < 1) {
```

Comment:
Hmm, then maybe this should instead be `if ((tabId && tabs.length <= 1) || tabs.length < 1)` - because we *don't* close any tabs when updating an existing snooze with a new time (which piggybacks on this `confirm` handler). That, or maybe I should add a separate `update` handler rather than re-using `confirm`.